src/__fixtures__/arrays/of-objects.json

```diff
@@ -8,6 +8,7 @@
       "type": ["array"],
       "items": {
         "type": "object",
+        "title": "SomeObj",
```

Comment:
```suggestion
```
unintended?

Reply:
Probably, from times when I was trying to make tests pass here. (Before changing direction and coming up with #111 ) 

Nice catch, thank you. Undone.